Header header
int8 total
float64[20] minAngle
float64[20] maxAngle